plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3SecurityMapping.java

```diff
@@ -70,13 +72,17 @@ public void testMapping()
         assertMapping(
                 provider,
                 path("s3://foo/data/test.csv"),
-                credentials("AKIAxxxaccess", "iXbXxxxsecret"));
+                credentials("AKIAxxxaccess", "iXbXxxxsecret")
+                        .withKmsKeyId("kmsKey_10")
+                        .withSseKmsKeyId("sseKmsKey_15"));
```

Comment:
It would be great if `S3SecurityMappingConfig` was created in the test instead of reading single uber fat `security-mapping.json` for all test cases.